experimental/examples/imageviewer/build.gradle.kts

```diff
@@ -15,5 +15,6 @@ allprojects {
         mavenCentral()
         maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
         mavenLocal()
+        maven("https://jitpack.io")
```

Comment:
We can't add jitpack dependencies because of potential security risks.
Also, we can't add this dependency as a jar, because it is not convenient.